app/Policies/ContactPolicy.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Policies;
+
+use App\Domain\Auth\Models\UserModel;
+use App\Domain\Contacts\Models\ContactModel;
+use Illuminate\Auth\Access\HandlesAuthorization;
+
+final class ContactPolicy
```

Comment:
I would store this class inside the `Domain` namespace, inside a sub folder `Policies`